### PR TITLE
[IMP] mail: introduce CallView/tileHeight

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2871878

### CODE REVIEW

File: addons/mail/static/src/components/call_view/call_view.js [hunk 1]
```diff
@@ -110,7 +109,7 @@ export class CallView extends Component {
         });
 
         this.state.tileWidth = tileWidth;
-        this.state.tileHeight = tileHeight;
+        this.callView.update({ tileHeight });
```
Comment (phenix-factory) [inline addons/mail/static/src/components/call_view/call_view.js:112]: You are using the private update for the with and the public update for the height.
You should use the public method if it's possible.

Comment (alexkuhn) [reply addons/mail/static/src/components/call_view/call_view.js:112]: This one in this PR is correct. The other PR that uses `_update()` is wrong.

**Comment by alexkuhn** [comment]
Comment point out problem in other PR. Is this PR good by itself?

**Review by phenix-factory — APPROVED** [review]
LGTM

**Comment by alexkuhn** [comment]
Moved to grouped PR: https://github.com/odoo/odoo/pull/93330

**Pull request closed without merging**